src/pages/scripts/tutorials/debug/index.md

```diff
@@ -13,10 +13,19 @@ contributors:
 This page goes over how to debug common problems you may run into.
 
 Quick tips for logs:
-* The UXP logs and `console.log` from scripts get written to the following locations: <br></br>
-    macOS: `/Users/<user name>/Library/Caches/UXPLogs/` <br></br>
-    Windows: `C:\Users\<username>\AppData\Local\Temp\UXPLogs` <br></br>
-    If you find multiple files in the location, pick the one with the latest timestamp.
+* The UXP logs and `console.log` from scripts get written to the following locations (For the InDesign versions below 19.2): <br></br>
```

Comment:
Please move the '(For the InDesign versions below 19.2)' to the next line so that developers can clearly distinguish the Id version numbers and the paths.

Reply:
Done